openhands/llm/llm.py

```diff
@@ -485,7 +512,10 @@ def get_token_count(self, messages):
         Returns:
             int: The number of tokens.
         """
-        return litellm.token_counter(model=self.config.model, messages=messages)
+        try:
+            return litellm.token_counter(model=self.config.model, messages=messages)
+        except Exception:
```

Comment:
Also raise a warning message here?

Reply:
Even though it's not used yet anywhere, I'd rather not add more spam to the log, just like the `Cost calculation not supported for this model.` warning did till we changed it.
Let me add a TODO in the code for this, if you're ok with that.